### PR TITLE
perf(Nuxt.js): optimize code

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Optimised the code
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/N/Nuxt.js/presence.ts [hunk 1]
```diff
@@ -1,70 +1,244 @@
...
+		await presence.getSetting<string>("lang").catch(() => "en")
+	);
+}
+function capitalizeFirstLetter(string: string) {
+	if (string) {
+		return (
+			string.trim().charAt(0).toUpperCase() +
+			string.trim().slice(1).toLowerCase()
+		);
+	} else return "Undefined";
```
Comment (Bas950) [inline websites/N/Nuxt.js/presence.ts:25]:
```suggestion
	if (!string) return "Undefined";
	return (
		string.trim().charAt(0).toUpperCase() +
		string.trim().slice(1).toLowerCase()
	);
```

File: websites/N/Nuxt.js/presence.ts [hunk 2]
```diff
@@ -1,70 +1,244 @@
...
+}
+function imgPath(path: string, hostname: string) {
+	if (path) {
+		if (path.includes(hostname)) return `https://${path.replace("//", "")}`;
+		else return `https://${hostname}${path}`;
+	} else return Assets.Logo;
```
Comment (Bas950) [inline websites/N/Nuxt.js/presence.ts:31]:
```suggestion
	if (!path) return Assets.Logo;
	if (path.includes(hostname)) return `https://${path.replace("//", "")}`;
	else return `https://${hostname}${path}`;
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/N/Nuxt.js/presence.ts (hunk 1 above)
Comment (EncryptedDev) [reply websites/N/Nuxt.js/presence.ts:25]: Still waiting for this one

**Review by Bas950 — APPROVED** [review]

File: websites/N/Nuxt.js/presence.ts [hunk 3]
```diff
@@ -1,70 +1,242 @@
...
+					details: "Viewing FAQs",
+				},
+				partners: {
+					details: "Viewing partners",
+				},
+				support: {
+					details: "Viewing the support page",
+				},
+				teams: {
+					details: "Viewing the nuxt.js team",
```
Comment (EncryptedDev) [inline websites/N/Nuxt.js/presence.ts:192]:
```suggestion
					details: "Viewing the Nuxt.js team",
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**